docker/px4-dev/Dockerfile_nuttx

```diff
@@ -26,10 +26,10 @@ RUN apt-get update \
 
 # GNU Arm Embedded Toolchain: 7-2017-q4-major December 18, 2017
 RUN mkdir -p /opt/gcc && cd /opt/gcc \
-	&& wget -qO- "https://armkeil.blob.core.windows.net/developer/Files/downloads/gnu-rm/7-2017q4/gcc-arm-none-eabi-7-2017-q4-major-linux.tar.bz2" | tar jx --strip 1 \
+	&& wget -qO- https://developer.arm.com/-/media/Files/downloads/gnu-rm/7-2017q4/gcc-arm-none-eabi-7-2017-q4-major-linux.tar.bz2 | tar jx --strip 1 \ 
 	&& rm -rf /opt/gcc/share/doc
 
-ENV PATH="$PATH:/opt/gcc/bin"
+ENV PATH="/opt/gcc/bin:$PATH"
```

Comment:
PATH is intentionally prepended for ccache.

Reply:
As the special fallthrough comments have to be parsed, I thought the problem was due to a mismatch between the pre-compiler and compiler that cmake finds, that is why I put gcc path first. Let me try without this.

Reply:
I think I'm missing something then. What are you seeing that happens with a compiler other than the one cmake finds?

Reply:
I can confirm that reverting this change brings the warning back.

Reply:
Here is the build log: https://pastebin.com/raw/Q8syW4RV

The only visible difference is the location of detected assembler: `/opt/gcc/bin/arm-none-eabi-gcc` vs `/usr/lib/ccache/arm-none-eabi-gcc`

Reply:
The warning is also back when ccache is prepended like this `ENV PATH="/usr/lib/ccache:/opt/gcc/bin:$PATH"`

Reply:
Can you try with `export CCACHE_RECACHE=1`.